q2cli/core/config.py

```diff
@@ -50,10 +52,23 @@ def _get_default_styles(self):
                 'required': {'underline': True},
                 'success': {'fg': 'green'}}
 
+    # This maintains the default colors while getting rid of all the default
+    # styling modifiers so what the user puts in their file is all they'll see
+    def get_editable_styles(self):
+        return {'option': {'fg': 'blue'},
+                'type': {'fg': 'green'},
+                'default_arg': {'fg': 'magenta'},
+                'command': {'fg': 'blue'},
+                'emphasis': {},
+                'problem': {'fg': 'yellow'},
+                'error': {'fg': 'red'},
+                'required': {},
+                'success': {'fg': 'green'}}
```

Comment:
I would make all of these empty, then if someone wants zero colorization, that's possible by giving an empty file as a theme.